network/servicenetwork/integr_test.go

```diff
@@ -156,14 +176,23 @@ func (s *testSuite) createNetworkNode(t *testing.T) networkNode {
 }
 
 func (s *testSuite) TestNodeConnect() {
```

Comment:
This test only covers case when discovery nodes connect to each other and form a new network. Do we plan to write a testcase in this task where a joiner node is connecting to an already formed network of discovery nodes?